### PR TITLE
GUACAMOLE-52: Document changes to history table schema

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: src/chapters/jdbc-auth.xml [hunk 1]
```diff
@@ -839,15 +839,32 @@ INSERT INTO guacamole_connection_parameter VALUES (<replaceable>1</replaceable>,
...
-                                connection being used.</para>
+                                connection being used. If the connection associated with the history
+                                record no longer exists, this will be
+                                <constant>NULL</constant>.</para>
+                        </listitem>
+                    </varlistentry>
+                    <varlistentry>
+                        <term><property>connection_name</property></term>
+                        <listitem>
+                            <para>The unique name associated with the connection at the time that it
```
Comment (jmuehlner) [inline src/chapters/jdbc-auth.xml]: Is the connection name actually unique?

Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml]: Yup. It isn't necessarily unique in the API, but it is in the schema, and that's enforced by a constraint.

Comment (jmuehlner) [inline src/chapters/jdbc-auth.xml]: Well it's not globally unique right? Only within the parent connection group.

Comment (mike-jumper) [inline src/chapters/jdbc-auth.xml]: True.

Very well. I shall document it ... *correctly*.

**Pull request merged**